spec/requests/graphql/mutations/eras/update_era_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe Mutations::Eras::UpdateEra, type: :request do
+  describe '.resolve' do
+    it 'updates an era' do
+      user = create(:user, :with_eras_events)
+      era_id = user.eras[0].id
+
+      post graphql_path, params: { query: query(era_id) }
+      result = JSON.parse(response.body)
+
+      data = result['data']['updateEra']
+
+      expect(data['name']).to eq('Master chambray tote bag fixie')
+      expect(data['userId']).to eq(user.id.to_s)
+      expect(data['startDate']).to eq('2021-01-04 00:00:00 UTC')
```

Comment:
Once Luke's PR on changing dates from DateTime to Date is merged, I can change these assertions to remove times.

Reply:
Looks Good!!

Reply:
Luke's PR has been merged!